src/css/styles.css

```diff
@@ -56,4 +56,39 @@ h2 {
   display: flex;
```

Comment:
We can probably create a single rule for `display: flex, align-items: center, justify-content: center` and stack the classes / elements that it should apply to.